features/toolbar/components/Toolbar.js

```diff
@@ -79,6 +80,7 @@ const mapDispatchToProps = dispatch => {
  * @static
  */
 Toolbar.propTypes = {
+    isVisible: PropTypes.bool.isRequired,
```

Comment:
```React.PropTypes``` is used below. So probably we should use one style.
Also we are sorting properties of object in alphabetical order.